appinventor/blocklyeditor/src/msg/en/_messages.js

```diff
@@ -1304,6 +1304,7 @@ Blockly.Msg.en.switch_language_to_english = {
     Blockly.Msg.REPL_NOT_NOW = "Not Now";
     Blockly.Msg.REPL_COMPANION_OUT_OF_DATE1 = "The Companion you are using is out of date.<br/><br/>This Version of App Inventor should be used with Companion version";
     Blockly.Msg.REPL_COMPANION_OUT_OF_DATE_IMMEDIATE = "You are using an out-of-date Companion. You should update the MIT AI2 Companion as soon as possible. If you have auto-update setup in the store, the update will happen by itself shortly.";
+    Blockly.Msg.REPL_COMPANION_WRONG_PACKAGE = "Make sure you are using the correct MIT AI2 Companion";
```

Comment:
Is it possible to always have the companion link there?   People are
already being sloppy about creating a companion at a find able place.
Can't we always stash a copy with the service?


== Hal Abelson
hal@mit.edu
Prof. of Comp. Sci. and Eng.
MIT Dept. of Elec. Eng. and Comp. Sci.



On Sat, Mar 12, 2016 at 3:53 PM, Jeffrey Schiller <notifications@github.com>
wrote:

> In appinventor/blocklyeditor/src/msg/en/_messages.js
> <https://github.com/mit-cml/appinventor-sources/pull/725#discussion_r55927475>
> :
>
> > @@ -1304,6 +1304,7 @@ Blockly.Msg.en.switch_language_to_english = {
> >      Blockly.Msg.REPL_NOT_NOW = "Not Now";
> >      Blockly.Msg.REPL_COMPANION_OUT_OF_DATE1 = "The Companion you are using is out of date.<br/><br/>This Version of App Inventor should be used with Companion version";
> >      Blockly.Msg.REPL_COMPANION_OUT_OF_DATE_IMMEDIATE = "You are using an out-of-date Companion. You should update the MIT AI2 Companion as soon as possible. If you have auto-update setup in the store, the update will happen by itself shortly.";
> > +    Blockly.Msg.REPL_COMPANION_WRONG_PACKAGE = "Make sure you are using the correct MIT AI2 Companion";
>
> @halatmit <https://github.com/halatmit> I'll make the change. There will
> always be some information under the Companion Information menu. If there
> isn't a companion available from the service itself, it will at least give
> you the version number. At the moment it doesn't give you the "name" of the
> companion to use. This is probably best left for the "branding" commits, as
> we have no way of knowing what the name of the companion is (we can learn
> its package name, but that can have no relation to the name given tot he
> App both on the device and in the store).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-cml/appinventor-sources/pull/725/files#r55927475>.
>